### PR TITLE
OGM-1471 Check Transaction status for Rollback

### PR DESCRIPTION
**Pull request by fax4ever**
https://hibernate.atlassian.net/browse/OGM-1471

### CODE REVIEW

File: core/src/test/java/org/hibernate/ogm/backendtck/queries/CompositeIdQueriesTest.java [hunk 1]
```diff
@@ -88,7 +88,8 @@ public void testSingleAttributeProjection() throws Exception {
 
 	@Test
 	public void testProjections() throws Exception {
-		List<ProjectionResult> result = asProjectionResults( session, "SELECT n.newsId.title, n.newsId.author, n.content FROM News n WHERE n.newsId.title = '" + ogmHowTo.getNewsId().getTitle() + "'" );
+		List<ProjectionResult> result = asProjectionResults(
+				session, "SELECT n.newsId.title, n.newsId.author, n.content FROM News n WHERE n.newsId.title = '" + ogmHowTo.getNewsId().getTitle() + "'" );
```
Comment (DavideD) [inline core/src/test/java/org/hibernate/ogm/backendtck/queries/CompositeIdQueriesTest.java]: Did you set the code auto-format? there several indentation that have changed. It's not critical but, please, refrain from doing it if you did it on purpose

**Comment by DavideD** [comment]
Half of the changes in this pull requests is code re-formatted. Could you remove those changes?

**Comment by fax4ever** [comment]
Sure @DavideD  I did it. Sorry for adding them.

**Comment by gsmet** [comment]
Merged, thanks!

**Comment by fax4ever** [comment]
Thank you!

**Pull request merged**